mava/systems/jax/system.py

```diff
@@ -72,11 +72,11 @@ def design(self) -> Tuple[DesignSpec, Dict]:
             Tuple[system callback components, system config].
         """
 
-    def update(self, component: Any) -> None:
-        """Update a component that has already been added to the system.
+    def update(self, components: Any) -> None:
+        """Update components that has already been added to the system.
 
         Args:
-            component: system callback component.
+            components: system callback component or list of components.
```

Comment:
nice! this makes things easier :smile:

Reply:
Glad you like it :slightly_smiling_face: